### ISSUE TITLE
How to sort when more than 10000 records in one resource

### ISSUE THREAD

**Issue by vFire** [ofcold/nova-sortable#4]
Will it save 10000 sort orders each time drag drop?

**Comment by lilianjin** [comment]
I don't quite understand what you are talking about.

**Comment by shaffe-fr** [comment]
Yes it will. Every rows of the current table will be sent and saved.
Everything is posted in a single field as base64, so you won't have a max_input_vars issue.
